usaspending_api/etl/es_award_template.json

```diff
@@ -408,30 +408,7 @@
           "type": "text"
         },
         "treasury_accounts": {
-          "type": "nested",
-          "properties": {
-            "aid": {
-              "type": "integer"
-            },
-            "ata": {
-              "type": "integer"
-            },
-            "main": {
-              "type": "integer"
-            },
-            "sub": {
-              "type": "integer"
-            },
-            "bpoa": {
-              "type": "integer"
-            },
-            "epoa": {
-              "type": "integer"
-            },
-            "a": {
-              "type": "text"
-            }
-          }
+          "type": "keyword"
```

Comment:
I'm surprised we can run regex on a keyword field based on [this help doc](https://www.elastic.co/guide/en/elasticsearch/reference/7.1/keyword.html)

![image](https://user-images.githubusercontent.com/22781949/74048624-d02d4480-498f-11ea-91b3-b6c187b8672c.png)

Reply:
If my understanding is correct that means that the `regexp` query will need to provide a regex that matches the entire string representation of the treasury_account instead of only a partial match. Struggling to find documentation that mentions exactly that.

Reply:
I would think this falls in the same boat as a wildcard query (e.g. `"wildcard": "uni*"`).
... Taking the `exact value` to mean the value provided in the source doc, and not any of the analyzed derivatives made. 
i.e. When not `keyword`, and using `text` with a "stemming" analyzer combined with "stop words" analyzer would take `"plays on his collegiate football team"` and then perform text searching on any of the terms `["play", "college", "football", "team"]`. 

Whereas use of `wildcard` or `regexp` on a `keyword` data type would be running matches on the exact value of the text as provided: `"plays on his collegiate football team"`

So I think it'll work. But it'd be good to have tests to verify.